index.dd

```diff
@@ -200,6 +200,31 @@ void main()
 }
 ----
 )
+$(EXTRA_EXAMPLE_RUNNABLE Tiny RPN calculator,
+$(RUNNABLE_EXAMPLE_STDIN 2 3 3 4 + * *)
+----
+void main()
+{
+    import std.stdio, std.string,
+        std.algorithm, std.conv, std.meta;
+
+    // Reduce the RPN expression using a stack
+    readln.split.fold!((stack, op)
```

Comment:
> Why do we need readln here?

Because a calculator program isn't very useful if it can calculate one single hard-coded expression.

> People don't need to switch tabs to browse stdin

I think this is an UI issue.

> We can take advantage of the fast, cached run.dlang.io backend

And this is definitely an implementation issue :)